### ISSUE TITLE
Is there any user guide or interface?

### ISSUE THREAD

**Issue by chrisli30** [NodeFactoryIo/vedran#191]
Hi guys, I think this project implements a great idea, because of two things:
1. In my opinion a centralized service, for example Infura is very expensive for a large request volume and not comfortible for me to solely depend on.
2. There are not many public hosted RPC provider for Polkadot ecosystem yet.

I'm still new to Polkadot, and walking through the Readme.md I've got two questions:
1. What does this project mean to an end user? How does him to pay for the requests, like to link his wallet to the load balancer?
2. Is there a user interface built for end users? For example if I'm using this load balancer in my iOS wallet app, where do I check the usage and payouts?

Thank you!

**Comment by mpetrunic** [comment]
> What does this project mean to an end user? How does him to pay for the requests, like to link his wallet to the load balancer?

- it allows users to run node on home network without port forwarding
- idea is that load balancer operator applies for reoccurring grant from polkadot/kusama treasury (still waiting for that to be implemented) and load banacer periodically pays on address which node operator provides

> Is there a user interface built for end users? For example if I'm using this load balancer in my iOS wallet app, where do I check the usage and payouts?

We currently only have grafana dashboard which load balancer may or may not publish ( http://westend.vedran.nodefactory.io:3000/?orgId=1&refresh=5s ) but it's interesting idea from some future improvements

**Comment by chrisli30** [comment]
> * reoccurring

Martin,

Thanks for the reply. Following up on your answers,

If I understand correclty, the public endpoint of the load balancer is an official URL from Polkadot/Kusuma? So the concept is very much alike Infura, which is funded by ecosystem fund and provides free service(to a certain degree) for convenience?

I see that there are multiple payout addresses in Grafana dashboard. Currently is it showing graphs for all node providers or just one behind the load balancer?

Thank you! Very interesting concept.

**Comment by mpetrunic** [comment]
Not sure what do you mean by official url?

Concept is somewhat like infura but idea is not to have single entity running bunch of nodes and charging for api usage. But to have multiple people which operate public load balancer and bunch of people running nodes and joining those load balancer pools where each gets paid for infrastructure it provides by chain itself.

> I see that there are multiple payout addresses in Grafana dashboard. Currently is it showing graphs for all node providers or just one behind the load balancer?

It's showing addresses of people which connected their node to load balancer

**Comment by chrisli30** [comment]
> But to have multiple people which operate public load balancer and bunch of people running nodes and joining those load balancer pools where each gets paid for infrastructure it provides by chain itself.

Yes, understood, but should there be an official list of load balancers? For example, when you host a load balancer user would need to connect to your URL, and when somebody else does it he/she will provide a different URL.

> It's showing addresses of people which connected their node to load balancer

Okay, got it. Thank you.

**Comment by mpetrunic** [comment]
> Yes, understood, but should there be an official list of load balancers? For example, when you host a load balancer user would need to connect to your URL, and when somebody else does it he/she will provide a different URL.

Yeah we will create website where people can register their loadbalancer and stats.
